pkg/services/alerting/notifiers/hipchat.go

```diff
@@ -123,15 +123,24 @@ func (this *HipChatNotifier) Notify(evalContext *alerting.EvalContext) error {
 
 	// Add a card with link to the dashboard
 	card := map[string]interface{}{
-		"style":       "link",
+		"style":       "application",
 		"url":         ruleUrl,
 		"id":          "1",
 		"title":       evalContext.GetNotificationTitle(),
-		"description": evalContext.GetNotificationTitle() + " in state " + evalContext.GetStateModel().Text,
```

Comment:
since you removed the GetStateModel().Text and it's not in the title, this will make the notification have info on state?

Reply:
There is still the [Alerting] and [OK] prefix in the title and the custom message is now there too.
I think this is sufficient

Reply:
yes, your right!